eurovision/notebooks/story.ipynb

```diff
@@ -993,7 +1043,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Average countries received "
+    "Before plotting the voting deviations, we have a look at the average score each country receives from each other country. The heatmap shows lighter colors for higher scores."
```

Comment:
"colors" or "colours"? I'm naturallly inclined towards en_gb, but en_us is also fine of course (I notice there are a couple of other uses elsewhere as well, all currently consistent with "colors").